src/main/java/com/imgix/URLHelper.java

```diff
@@ -198,9 +198,6 @@ public static String sanitizePath(String path) {
       // Use encodeURIComponent to ensure *all* characters are handled,
       // since it's being used as a path
       return "/" + URLHelper.encodeURIComponent(path);
-    } else if (path.startsWith("http%3A") || path.startsWith("https%3A")) {
-      // To nothing to the URL being used as path since already encoded
-      return "/" + path;
```

Comment:
so should we add this back in since it was previously supported behavior?
Apologies for going in circles on this 😅

Reply:
Shoot yup that's my bad. The stacked PR addresses this but to be safe those lines should be added back in here as well.